suricata.yaml.in

```diff
@@ -158,6 +158,8 @@ outputs:
       #  pipelining:
       #    enabled: yes ## set enable to yes to enable query pipelining
       #    batch-size: 10 ## number of entry to keep in buffer
+      # Log ethernet header in events when available
+      #log-ethernet: yes
```

Comment:
@jasonish it is not at the same level (not under alert) so I was thinking about being more explicit.